test/e2e/specs/wp-signup-spec.js

```diff
@@ -1763,7 +1763,9 @@ describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function() {
 		} );
 	} );
 
-	describe( 'Passwordless signup @parallel', function() {
+	// Disable test while Passwordless functionality is completely switched off
+	// https://github.com/Automattic/wp-calypso/pull/37054
+	describe.skip( 'Passwordless signup @parallel', function() {
```

Comment:
Seems unrelated?

Reply:
Yup, it seems like it got there in this commit: https://github.com/Automattic/wp-calypso/pull/37510/commits/469ee000da56853cfa99bb3a4dad54f95f1bff1f

I'm going to remove it

Reply:
cc @p-jackson

Reply:
Yes I did it to get CI passing. Rebasing the branch would have been the less confusing thing to do.